README.md

```diff
@@ -1,46 +1,159 @@
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+<!-- LIVE DEMO -->
 
-## Learn More
+## 🚀 Live Demo <a name="live-demo"></a>
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- [Live Demo Link](https://yuvenalmash.github.io/stock-411/)
+- [video Demo](https://www.loom.com/share/c8c09a827ec041dfb0827a55438a2b51)
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ $ sudo apt install node
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  $ cd my-folder
+  $ git clone https://github.com/yuvenalmash/stock-411.git
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  $ npm install
+  $ npm run start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  $ npm run test
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Yuvenal Njoroge**
+
+- GitHub: [@yuvenalmash](https://github.com/yuvenalmash)
+- Twitter: [@_Juvenal_](https://twitter.com/_Juvenal)
+- LinkedIn: [Yuvenal Njoroge](https://linkedin.com/in/yuvenal-njoroge)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Add more categories]**
+- [ ] **[Add a page offering visualised data]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/yuvenalmash/stock-411/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Nelson Sakwa on behance](https://www.behance.net/sakwadesignstudio) for the design inspiration.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly add an MIT license to your project. Currently, when I click on the MIT license link, it returns a 404 page. Please add your MIT license to this project. 👍🏽